<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpDialog</class>
 <widget class="QDialog" name="SignUpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Script</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: white;
}</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_background_signup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>531</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="SignupButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>560</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
 color: white;
   background-color: #887f94;
    border: 2px solid black; /* تنظیم رنگ و ضخامت خط دور */
    padding: 5px; /* تنظیم فاصله متن از لبه‌ها */

  border-radius: 10px;
    padding: 10px;
    font-size: 16px;
}

</string>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>441</width>
     <height>351</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="NamelineEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid #ccc;
    border-radius: 10px;
    padding: 10px;
    font-size: 14px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LastNamelineEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid #ccc;
    border-radius: 10px;
    padding: 10px;
    font-size: 14px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Last name</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="PhonelineEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid #ccc;
    border-radius: 10px;
    padding: 10px;
    font-size: 14px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Phone number</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="UserNamelineEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid #ccc;
    border-radius: 10px;
    padding: 10px;
    font-size: 14px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Create username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PasswordlineEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid #ccc;
    border-radius: 10px;
    padding: 10px;
    font-size: 14px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Create password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PassConfirmlineEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid #ccc;
    border-radius: 10px;
    padding: 10px;
    font-size: 14px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Confirm password</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>620</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">QLabel{
    color: #bcafbc;
    cursor: pointer;
}</string>
      </property>
      <property name="text">
       <string>Already have an account?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="LoginButton">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    color: #00f;
    background-color: transparent;
    border: none;
    text-decoration: underline;
    cursor: pointer;
}
</string>
      </property>
      <property name="text">
       <string>log in</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>690</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
